v0100/smlrc.c

```diff
@@ -2886,7 +2890,7 @@ int exprUnary(int tok, int* gotUnary, int commaSeparator, int argOfSizeOf)
       {
         tok = GetToken();
         tok = expr(tok, gotUnary, 0);
-        if (!*gotUnary)
+        if (!*gotUnary || tok != ']')
```

Comment:
Why this change?

Reply:
This change ended up in the final Commit, it was part of a unification test with the next IF, to decrease the final size of the executable, can be restored to the previous state.